src/main/kotlin/db/migration/data/dev/R__2_5_Integration_test_data.kt

```diff
@@ -25,29 +27,116 @@ import java.util.UUID
 @Suppress("ClassName", "unused")
 class R__2_5_Integration_test_data : BaseJavaMigration() {
 
+  private val logger = LoggerFactory.getLogger(javaClass)
+
   override fun migrate(context: Context) {
     JdbcTemplate(SingleConnectionDataSource(context.connection, true)).also {
       createStandardMoves(it)
+      createRedirectMoves(it)
+      createLongHaulMoves(it)
+      createLockoutMoves(it)
+      createMultiTypeMoves(it)
+      createCancelledMoves(it)
     }
   }
 
   private fun createStandardMoves(template: JdbcTemplate) {
+    logger.info("create standard moves")
+
     mapOf(
       "SM1" to "PR1",
       "SM2" to "PR2",
       "SM3" to "PR3"
     ).forEach {
-      create(move(moveId = it.key, profileId = it.value, fromAgencyId = "SFROM", toAgencyId = "STO"), template).also { move ->
+      create(move(moveId = it.key, profileId = it.value, fromAgencyId = "FROM_AGENCY", toAgencyId = "TO_AGENCY"), template).also { move ->
         create(moveStartEvent(move), template)
         create(moveCompleteEvent(move), template)
-        create(journey(move, fromAgencyId = "SFROM", toAgencyId = "STO"), template).also { journey ->
+        create(journey(move, fromAgencyId = "FROM_AGENCY", toAgencyId = "TO_AGENCY"), template).also { journey ->
           create(journeyStartEvent(journey), template)
           create(journeyCompleteEvent(journey), template)
         }
       }
     }
   }
 
+  // TODO need to add the appropriate journey events for the following move types...
```

Comment:
This TODO does not actually affect the integration test.  It is there to remind me I need to do it!